routes/mock-courses.json

```diff
@@ -0,0 +1,302 @@
+[
+  {
+    "id": "1",
+    "study_abroad_institution": "Curtin University",
```

Comment:
Let's change all the keys to `camelCase` and later when we start to deal with `MySQL`, we will write an utility function that will transform data into `camelCase` and back to `snake_case`, and vice versa later so that frontend can only deal with `camelCase` instead of all the other different casing that may arise later down the road.